share/pvss/pvss.go

```diff
@@ -15,13 +15,28 @@ package pvss
 
 import (
 	"errors"
+	"hash"
 
 	"github.com/dedis/crypto"
 	"github.com/dedis/crypto/proof"
-	"github.com/dedis/crypto/util/random"
 	"github.com/dedis/crypto/share"
+	"github.com/dedis/crypto/util/random"
 )
 
+// NOTE. Here, we need need the full suite definition for using the proof/
+// package while we only use Hash.
```

Comment:
Isn't this because you haven't yet narrowed down the set of members that proof.Suite actually needs to have?  Of course this Suite interface will be unnecessarily broad until/unless those of all the modules it depends on have been slimmed down appropriately as well.  I don't see a methodology problem here, only an (understandable) problem that the methodology hasn't been fully implemented yet.